app/src/components/market/market_fund_container.tsx

```diff
@@ -10,10 +10,10 @@ interface Props {
   marketMakerAddress: string
 }
 
-const MarketFundContainer: FC<Props> = props => {
+const MarketFundContainer = (props: Props) => {
```

Comment:
`FC<Props>` gives better type guarantees, any reason for this change?

Reply:
That change was to maintain consistency with the declaration of the other similar containers files, I will add it there and in the other files